src/js/utils/merge-options.js

```diff
@@ -41,7 +43,7 @@ const customizer = function(destination, source) {
  * provided objects
  * @function mergeOptions
  */
-export default function mergeOptions() {
+mergeOptions = function() {
```

Comment:
We can't because `customizer` uses `mergeOptions` and `mergeOptions` uses `customizer`. Perhaps a better solution is to remove the rule that says you can't use a function before it's defined (which is a silly rule unless you explicitly forbid function declarations, which we're no longer doing).